src/main/java/org/sasanlabs/service/vulnerability/csrf/CSRFVulnerability.java

```diff
@@ -0,0 +1,218 @@
+package org.sasanlabs.service.vulnerability.csrf;
+
+import java.io.IOException;
+import java.util.Random;
+import java.util.function.Supplier;
+import org.apache.commons.lang3.StringUtils;
+import org.sasanlabs.internal.utility.LevelConstants;
+import org.sasanlabs.internal.utility.Variant;
+import org.sasanlabs.internal.utility.annotations.AttackVector;
+import org.sasanlabs.internal.utility.annotations.VulnerableAppRequestMapping;
+import org.sasanlabs.internal.utility.annotations.VulnerableAppRestController;
+import org.sasanlabs.service.vulnerability.bean.GenericVulnerabilityResponseBean;
+import org.sasanlabs.vulnerability.types.VulnerabilityType;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+/**
+ * This class contains vulnerabilities related to CSRF.
+ *
+ * @author xxx
+ */
+@VulnerableAppRestController(descriptionLabel = "CSRF_VULNERABILITY", value = "CSRFVulnerability")
+public class CSRFVulnerability {
+
+    private static final String RTN_BADINPUT = "Bad input!";
+
+    private static final String USR_LOGIN = "id";
+    private static final String USR_PASS = "pass";
+    private static final String USR_NEWPASS = "newpass";
+
+    private static final String LOGIN = "mylogin";
+    private String passlive = "mypassword";
```

Comment:
Use SHA256 + hmac with salt (random), a hash of the password instead of a plain text password. Why? because we also support [SonarSource](https://sonarsource.com/). 
I would suggest adding one more level where put the MD5 hash so that we can expect SonarSource to find the vulnerability.